components/Mypage/successModal.tsx

```diff
@@ -27,32 +25,19 @@ interface SuccessInfo {
 export default function SuccessModal({ ...props }: SuccessInfo) {
   const router = useRouter();
   const toast = useToast();
+  const [open, setOpen] = useState(props.success);
 
   function handleSuccess() {
-    const toastId = 'success';
-    if (!toast.isActive(toastId)) {
-      toast({
-        id: toastId,
-        status: 'success',
-        description:
-          '업체 등록이 성공적으로 신청되었어요. 다시 로그인 해주세요.',
-        duration: 1000,
-        isClosable: true,
-        containerStyle: {
-          marginBottom: '60px',
-        },
-      });
-    }
-    props.setAllSuccessFun(true);
+    toast({
+      status: 'success',
+      description: '업체 등록이 성공적으로 신청되었어요. 다시 로그인 해주세요.',
+      isClosable: true,
+    });
```

Comment:
```suggestion
     const toastId = 'success';
    if (!toast.isActive(toastId)) {
      toast({
        id: toastId,
        status: 'success',
        description:
          '업체 등록이 성공적으로 신청되었어요. 다시 로그인 해주세요.',
        duration: 1000,
        isClosable: true,
        containerStyle: {
          marginBottom: '60px',
        },
      });
    }
```

이부분은 중복생성 방지랑 위치통일로 변경되었어요~